.github/ISSUE_TEMPLATE/propose_new_rule.yml

```
@@ -0,0 +1,88 @@
+name: Propose a new rule
+description: Propose a new rule for the eslint-plugin-testing-library.
+labels: [new rule]
+body:
+  - type: input
+    id: name_for_new_rule
+    attributes:
+      label: Name for new rule
+      description: Suggest a name for the new rule that follows the [rule naming conventions](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md#rule-naming-conventions).
+      placeholder: prefer-wait-for
+    validations:
+      required: true
+
+  - type: textarea
+    id: description_of_the_new_rule
+    attributes:
+      label: Description of the new rule
+      description: Please describe what the new rule should do.
+      placeholder: The rule should ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: testing_library_feature
+    attributes:
+      label: Testing Library feature
+      description: What Testing Library feature does this rule relate to?
+      placeholder: Enforce promises from async queries to be handled
+    validations:
+      required: true
+
+  - type: textarea
+    id: testing_library_frameworks
+    attributes:
+      label: Testing Library framework(s)
+      description: What Testing Library framework(s) does this rule relate to?
+      placeholder: angular, react and vue
```

```suggestion
      placeholder: Angular, React and Vue
```

We should probably capitalize these names. 😄

---

Sure!